mobly/controllers/android_device_lib/services/snippet_management_service.py

```diff
@@ -37,6 +38,16 @@ def __init__(self, device, configs=None):
     self._is_alive = False
     self._snippet_clients = {}
     super().__init__(device)
+    self._is_using_client_v2 = None
+
+  @property
+  def is_using_client_v2(self):
```

Comment:
can we name this the "private" way since we probably don't want users to call this.

Reply:
Done.